.env.example

```diff
@@ -110,6 +110,9 @@ GITHUB_TOKEN=
 
 
 ## Jenkins, optional, for triggering Jenkins builds after deployment
+## When JENKINS_NOTIFY_COMMITTERS is set, it will send out emails of deployer, buddy-approver ( if available ), and committers
+## to jenkins as "emails" param. If GOOGLE_DOMAIN is set, then emails are filtered by domain.
 # JENKINS_URL= # server_url of jenkins
 # JENKINS_USERNAME= # user id
 # JENKINS_API_KEY= # API Token from user / Configure page
+# JENKINS_NOTIFY_COMMITTERS=1 #optional
```

Comment:
@grosser that's related one of the two questions that we raised in the description:

> Should there be corresponding UI changes for sending emails of committers. Presently, if JENKINS_NOTIFY_COMMITTERS is set, it will send email of committers to jenkins, otherwise not.

Do we want to make this as a checkbox on the staging configuration page so that owners of different stages could decide whether they want to notify committers and/or deployer/buddy? Or are we good with starting with a global option.

Of course, it's still up to the jenkins builds to configure whether this new `emails` build parameter will be used to send email notifications.